words.yaml

```diff
@@ -1613,13 +1619,434 @@ zyn:
   _family: R
   _signature: "..[1,2]"
   eng_long: ___ is greater than ___ in property [___1,2]
   eng_short: greater
-plen:
+OLD_plen:
   _family: R
   _signature: "."
   eng_long: ___ is rain
   eng_short: rain
+
+tan:
+  _family: R
+  _signature: "00"
+  eng_long: "[___] (default = now) is in the past of/before [___] (default = now)"
+  eng_short: past
+tar:
+  _family: R
+  _signature: "00"
+  eng_long: "[___] (default = now) is in the present of/concurrent with [___] (default = now)"
+  eng_short: present
+tal:
+  _family: R
+  _signature: "00"
+  eng_long: "[___] (default = now) is in the future of/after [___] (default = now)"
+  eng_short: future
+
+tahan:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is forward/in front of [___X] (default = here)"
+  eng_short: forward
+tahen:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is backward of/behind [___X] (default = here)"
+  eng_short: backward
+tahin:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is leftward/to the left of [___X] (default = here)"
+  eng_short: left
+tahon:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is rightward/to the right of [___X] (default = here)"
+  eng_short: right
+tahun:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is downward of/under [___X] (default = here)"
+  eng_short: down
+tahyn:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is upward of/above [___X] (default = here)"
+  eng_short: up
+
+tahar:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is northward/to the north of [___X] (default = here)"
+  eng_short: north
+taher:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is southward/to the south of [___X] (default = here)"
+  eng_short: south
+tahir:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is westward/to the west of [___X] (default = here)"
+  eng_short: west
+tahor:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is eastward/to the east of [___X] (default = here)"
+  eng_short: east
+
+tahal:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is within/inside [___X] (default = here)"
+  eng_short: inside
+tahel:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is surrounding/orbiting/outside [___X] (default = here)"
+  eng_short: outside
+tahil:
+  _family: R
+  _signature: "[X][X]"
+  eng_long: "[___X] (default = here) is passing through/transfixing [___X] (default = here)"
+  eng_short: through
+
+tanan:
+  _family: R
+  _signature: "00"
+  eng_long: "[___] (default = now) is at a negligible distance in time of [___] (default = now)"
+  eng_short: negligible time distance
+tanar:
+  _family: R
+  _signature: "00"
+  eng_long: "[___] (default = now) occurs in a negligible interval of [___] (default = now)"
```

Comment:
nit - maybe we should write "negligible time interval" or "negligible interval of time" (instead of just "negligible interval")?